### PR TITLE
Revert prozzie udp ports

### PR DESCRIPTION
**Pull request by Chorro**

### CODE REVIEW

File: cli/include/config.bash [hunk 1]
```diff
@@ -421,6 +421,9 @@ zz_enable_disable_modules() {
             ;;
         esac
     done
+
```
Comment (eugpermar) [inline cli/include/config.bash]: Could this be in a separate commit?

**Comment by eugpermar** [comment]
Hi! Could http2k have it own testing of enable/disable?

File: tests/tests_config01.bash [hunk 2]
```diff
@@ -351,12 +359,21 @@ testEnableModule() {
...
 
     if [[ -L "${PROZZIE_PREFIX}/etc/prozzie/compose/f2k.yaml" ]]; then
         ${_FAIL_} '"prozzie config --disable must to unlink f2k compose file"'
     fi
 
+    if [[ -L "${PROZZIE_PREFIX}/etc/prozzie/compose/monitor.yaml" ]]; then
+        ${_FAIL_} '"prozzie config --disable must to unlink monitor compose file"'
+    fi
+
+    snmptrap -v 2c -c public ${HOSTNAME} "" 1.3.6.1.4.1.2021.13.991 .1.3.6.1.2.1.1.6 s "Device in Wizzie"
```
Comment (eugpermar) [inline tests/tests_config01.bash]: Should this command fail? (i.e., return code should be different than 0, isn't?)

File: compose/f2k.yaml [hunk 3]
```diff
@@ -7,7 +7,7 @@ services:
     depends_on:
       - kafka
     ports:
-      - 2055:2055
+      - 2055:2055/udp
```
Comment (eugpermar) [inline compose/f2k.yaml]: Could this be in a separated commit?

File: tests/ubuntu-16.04/Dockerfile [hunk 4]
```diff
@@ -122,11 +122,13 @@ ARG PSLIST
 ARG TCL
 ARG EXPECT
 ARG TCLLIB
+ARG SNMPTRAP
 
 RUN ! ${IS_CENTOS} || ${PKG_MANAGER_INSTALL} epel-release
 RUN ${PKG_MANAGER_UPDATE} && ${PKG_MANAGER_INSTALL} \
 	${EXPECT} \
 	${KAFKACAT} \
+    ${SNMPTRAP} \
```
Comment (eugpermar) [inline tests/ubuntu-16.04/Dockerfile]: Could this be indented by tab, in order to homogenize its indentation with others packages?

Comment (Chorro) [reply tests/ubuntu-16.04/Dockerfile]: Ok

File: compose/f2k.yaml (hunk 3 above)
Comment (Chorro) [reply compose/f2k.yaml]: Ok

File: tests/tests_config01.bash (hunk 2 above)
Comment (Chorro) [reply tests/tests_config01.bash]: I'm going to check it 👍

**Comment by Chorro** [comment]
Ok. All done! 👍
